test/services/api/v1/issues/filter_test.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class Api::V1::Issues::FilterTest < ActiveSupport::TestCase
+  setup do
+    Current.account = Account.first
```

Comment:
```suggestion
    Current.account = send 'public.accounts', :default
```
Aunque siempre debería devolver la misma, mejor si lo dejamos bien explícito por las dudas.

Reply:
dale franco me parece perfecto